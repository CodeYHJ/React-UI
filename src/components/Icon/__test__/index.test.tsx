import { Icon } from "../index"
import React from "react"
import renderer from "react-test-renderer";
describe('Icon', () => {
    it('renders an `.code-ui-svg` component', () => {
        const json = renderer.create(<Icon name="main" />)
        expect(json).toMatchSnapshot()
    })
})