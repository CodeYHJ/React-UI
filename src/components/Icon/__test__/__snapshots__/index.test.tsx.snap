// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon renders an \`.code-ui-svg\` component 1`] = `
<svg
  className="code-ui-svg"
>
  <use
    xlinkHref="#main"
  />
</svg>
`;
